### PR TITLE
asciinema: automatically generate recordings

### PR DESCRIPTION
**Pull request by miampf**
Automatically generate asciinema recordings for the emojivoto demo. A successful CI run can be found [here](https://github.com/edgelesssys/contrast/actions/runs/9893811646/job/27329861438) (it *technically* fails because the generated PR is based on main which causes merge conflicts. This will be resolved after merging this PR).

### CODE REVIEW

**Comment by katexochen** [comment]
Please rebase on 252aab26cb866e064f5d8d9ae42e1154c7d32db3

File: tools/asciinema/window-frame.svg
Comment (katexochen) [inline tools/asciinema/window-frame.svg:1]: Is this used anywhere? Otherwise I think we can remove it.

Comment (katexochen) [inline tools/asciinema/window-frame.svg:1]: Removal was sqashed into the wrong commit, should go into the first one.

File: .github/workflows/asciinema.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      - name: Create justfile.env
+        run: |
+          cat <<EOF > justfile.env
+          container_registry=${{ env.container_registry }}
+          azure_resource_group=${{ env.azure_resource_group }}
+          EOF
+      - name: Get credentials for CI cluster
+        run: |
+          just get-credentials
+      - name: Get tickte for the CI cluster
```
Comment (m1ghtym0) [inline .github/workflows/asciinema.yml:43]:
```suggestion
      - name: Get ticket for the CI cluster
```

**Review by m1ghtym0 — APPROVED** [review]
Thanks, LGTM

**Pull request merged**